R/global_sessions_config.R

```diff
@@ -29,6 +29,8 @@
 #' allow user to do certain actions (such as viewing charts and tables) without signing in,
 #' and only require users to sign in if they want to save data to your database.
 #' @param sentry_dsn either \code{NULL}, the default, or your Sentry project DSN.
+#' @param cookie_expiration the number of days before a User's cookie expires.
+#' Set to `NULL` to force Sign Out at session end. Default is `365` (i.e. 1 year)
```

Comment:
use \code{NULL} not `NULL`

Reply:
c0629b28e77a52a0cb52ba35abf2f0ea50cfef95